src/wallet/rpczerocoin.cpp

```diff
@@ -1365,6 +1365,7 @@ UniValue showspendcaching(const JSONRPCRequest& request)
     }
 
     auto ztracker = pwallet->GetZTrackerPointer();
+    LOCK(ztracker->cs_readlock);
```

Comment:
What makes this one need the lock, and other callers to ListMints not?

Reply:
only this one calls GetSpendCache() which needs the lock.
but tbh this is going away in another commit anyhow.

Reply:
another PR*